src/contracts/atlas/Mimic.sol

```diff
@@ -47,7 +47,11 @@ contract Mimic {
                 bytes32(uint256(0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee))
             )
         );
-        if (!success) revert();
+        if (!success) {
+            assembly {
+                revert(add(output, 32), mload(output))
+            }
+        }
```

Comment:
@thogard785 - Changed this to bubble up the revert reason. Needed for unit tests.